src/preset_cli/cli/main.py

```diff
@@ -629,9 +601,9 @@ def sync_user_role_to_workspace(
     )
 
 
-preset_cli.add_command(auth)
-preset_cli.add_command(invite_users)
-preset_cli.add_command(import_users)
+preset_cli.add_command(auth, name="auth")
+preset_cli.add_command(invite_users, name="invite-users")
+preset_cli.add_command(import_users, name="import-users")
```

Comment:
For some reason, for the commands that involved the `raise_cli_errors` decorator, the tests were failing with command not found until I explicitly added the command names. 🤷